concepticondata/conceptlists/Gauchat-1925-480.tsv

```diff
@@ -0,0 +1,481 @@
+Gauchat-1925-480-161	161	the ewes	les brebis	fētas	54	1345	EWE
+Gauchat-1925-480-162	162	sharpen (?)	racérer	reaciariare	54		
+Gauchat-1925-480-163	163	the axe	la hache	happia (+ itta)	56	677	AXE
+Gauchat-1925-480-164	164	beat in the barn	battre en grange	excŭtere	56		
+Gauchat-1925-480-165	165	grind	moudre	mŏlere	56	1033	GRIND
+Gauchat-1925-480-166	166	the wheat	le blé	blatu	56	1077	WHEAT
+Gauchat-1925-480-167	167	keep	garder	wardare	56	1477	KEEP
+Gauchat-1925-480-168	168	the livestock	les bêtes	bĕstias	56	631	LIVESTOCK
+Gauchat-1925-480-169	169	turn	tourner	(re)virare	58	1588	TURN
+Gauchat-1925-480-170	170	the earth	la terre	tĕrra	58	1228	EARTH (SOIL)
+Gauchat-1925-480-171	171	to sow	semer	seminare	58	748	SOW SEEDS
+Gauchat-1925-480-172	172	(the) hemp	(le) chanvre	cannapu	58	2041	HEMP
+Gauchat-1925-480-173	173	the beans	les fèves	fabas	58	832	BEAN
+Gauchat-1925-480-174	174	The horse	Le cheval	caballu	58	615	HORSE
+Gauchat-1925-480-175	175	is sold	est vendu	vendūtu	60		
+Gauchat-1925-480-176	176	The cow	La vache	vacca	60	1007	COW
+Gauchat-1925-480-177	177	(is) sold (feminine)	(est) vendue	vendūta	60		
+Gauchat-1925-480-178	178	The goats	Les chèvres	capras	60	1502	GOAT
+Gauchat-1925-480-179	179	(are) sold	(sont) vendues	vendūtas	60		
+Gauchat-1925-480-180	180	It is expensive	Il est cher	caru	60		
+Gauchat-1925-480-181	181	(she is) expensive	(Elle est) chère	cara	62		
+Gauchat-1925-480-182	182	(they are) expensive	(elles sont) chères	caras	62		
+Gauchat-1925-480-183	183	The pig	Le porc	pŏrcu	62	1337	PIG
+Gauchat-1925-480-184	184	is fat	est gros	grŏssu	62	2449	FAT (OF MEAT AND FOOD)
+Gauchat-1925-480-185	185	The sow	La truie	trŏia	62	1351	SOW (FEMALE PIG)
+Gauchat-1925-480-186	186	(is) fat (feminine)	(est) grosse	grŏssa	62		
+Gauchat-1925-480-187	187	The hay	Le foin	fēnu	64	342	HAY
+Gauchat-1925-480-188	188	is dry	est sec	sĭccu	64	1398	DRY
+Gauchat-1925-480-189	189	The grass	L’herbe	hĕrba	64	606	GRASS
+Gauchat-1925-480-190	190	(is) dry (feminine)	(est) sèche	sĭcca	64		
+Gauchat-1925-480-191	191	What do you want?	Que veux-tu?	quĭd vŏles tū	64		
+Gauchat-1925-480-192	192	I would like	Je voudrais	volere + habēbam	64		
+Gauchat-1925-480-193	193	some salt	du sel	sal	66	1274	SALT
+Gauchat-1925-480-194	194	some pepper	du poivre	pĭper	66	1385	PEPPER
+Gauchat-1925-480-195	195	one leaf	une feuille	fŏlia	66	628	LEAF
+Gauchat-1925-480-196	196	some cabbage	de chou	caule	66	774	CABBAGE
+Gauchat-1925-480-197	197	a wreath	une gerbe	garba	66	2589	WREATH
+Gauchat-1925-480-198	198	some straw	de paille	palea	66		
+Gauchat-1925-480-199	199	a bag	un sac	saccu	68	2468	BAG
+Gauchat-1925-480-200	200	some oat	d’avoine	avēna	68	60	OAT
+Gauchat-1925-480-201	201	some flour	de farine	farina	68	1594	FLOUR
+Gauchat-1925-480-202	202	an armful	une brassée	braciata	68		
+Gauchat-1925-480-203	203	some armfuls	(des) brassées	braciatas	68		
+Gauchat-1925-480-204	204	of wood	de bois	bŏscu	68	1803	WOOD
+Gauchat-1925-480-205	205	(a) shovel	(une) pelletée	palata	70		
```

Comment:
1901 SHOVEL

Reply:
Ah there's already another SHOVEL above - dunno which fits better.

Reply:
The French pelletée refers more to quantity, comparable to 'a handful' or 'a spoonful'. The context here would then be something like 'a spadeful of coal', I guess.